README.md

```diff
@@ -28,34 +28,105 @@ The ```winetricks``` package should be used if it is available and up to date. T
 ```
-sudo -- sh -c '
-cat > /usr/local/bin/update_winetricks.sh <<_EOF_SCRIPT
+
+Then, for Ubuntu, use a simple shell script to download the current winetricks scripts.
+This script uses a single parameter (to set a working directory).
+E.g.:
+
+```
+# Change to a directory writeable by current user (their HOME directory). See:
+#   https://www.tldp.org/LDP/abs/html/internalvariables.html
+cd "${HOME}"
+
+# Use a BASH "here document" to create an updater shell script file.
+# See:
+#   https://www.tldp.org/LDP/abs/html/here-docs.html
+# >  outputs stdout to a file, overwriting any pre-existing file
+# << takes input, directly from the script itself, till the second '_EOF_SCRIPT' marker, as stdin
+# the cat command hooks these 2 streams up (stdin and stdout)
+###### create update_winetricks.sh START ########
+cat > update_winetricks.sh <<_EOF_SCRIPT
 #!/bin/sh
 
-cd "${HOME}" # change to a path writeable by current user
+# Change to a directory writeable by current user. See:
+#   https://www.tldp.org/LDP/abs/html/internalvariables.html
+# Use a script parameter $1 or fallback to the user's HOME directory.
+# This works around the lack of env variable support in systemd units (see below).
+cd "\${1:-\${HOME}}"
+
+# Download the latest winetricks script (master="latest version") from Github.
 wget https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
+
+# Mark the winetricks script (we've just downloaded) as execuable. See:
+#   https://www.tldp.org/LDP/GNU-Linux-Tools-Summary/html/x9543.htm
 chmod +x winetricks
-sudo mv winetricks /usr/local/bin
+
+# Move the winetricks script to a location which will be in the standard user PATH. See:
+#   https://www.tldp.org/LDP/abs/html/internalvariables.html
+sudo mv winetricks /usr/bin
+
+# Download the latest winetricks BASH completion script (master="latest version") from Github.
 wget https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks.bash-completion
+
+# Move the winetricks BASH completion script to a location standard location for BASH completion modules. See:
+#   https://www.tldp.org/LDP/abs/html/tabexpansion.html
 sudo mv winetricks.bash-completion /usr/share/bash-completion/completions/winetricks
-_EOF_SCRIPT
-chmod +x /usr/local/bin/update_winetricks.sh
+_EOF_SCRIPT 
+###### create update_winetricks.sh FINISH ########
+
+# Mark the update_winetricks script (we've just written out) as execuable. See:
+#   https://www.tldp.org/LDP/GNU-Linux-Tools-Summary/html/x9543.htm
+chmod +x update_winetricks.sh
+
+# We must escalate privileges, to root, as regular Ubuntu users do not normally have write access to '/usr/bin'.
+sudo mv update_winetricks.sh /usr/bin/
 '
+```
 
+See the manpages for the individual functions, if you are not clear how they are being used, e.g.
+```
+man wget
+man sudo
+...
+```
+
+For Linux distributions that do not have **sudo** available by default, then use this shell script variant to download the current winetricks scripts:
```

Comment:
What distros are those? Or are you just being safe?

Reply:
That was a joke about one of the recent winetricks pull requests... :-)
I'll re-phrase that.